README.md

```diff
@@ -45,8 +45,17 @@ The app targets shoppers across Australia who are looking for unique and quality
 - Japanese food products lovers
 
 ## User Stories 
+We construct user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying potential users' unique needs and expectations, including a system administrator at Sakura company.
```

Comment:
Just a few minor spelling and formatting mistakes here and there. I suggest paste the paragraph to Word and spell check first before pasting back to Markdown.